packages/notifi-react-card/lib/components/subscription/NotifiSubscriptionContextWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  NotifiSubscriptionContextProvider,
+  useNotifiClientContext,
+} from 'notifi-react-card/lib';
+import React from 'react';
+
+import { NotifiInputLabels, NotifiInputSeparators } from '.';
+
+export type NotifiSubscriptionContextWrapper = Readonly<{
+  inputLabels?: NotifiInputLabels;
+  darkMode?: boolean;
```

Comment:
i do think these props will be helpful to use in other components as we build but not sure how it will be for our current integration

Reply:
Can you please elaborate on this more? I was curious as to why we may still be typing the wrapper the same way that we would type the SubscriptionCard. 

Is the idea that integrators will eventually pass in their own type of card?